security/jwt/src/main/java/io/helidon/security/jwt/Jwt.java

```diff
@@ -226,7 +228,11 @@ public class Jwt {
         this.payloadClaims = getClaims(payloadJson);
 
         // known payload
-        this.issuer = JwtUtil.getString(payloadJson, "iss");
+        if (payloadJson.get("iss") instanceof JsonString) {
+            this.issuers = JwtUtil.getString(payloadJson, "iss").map(List::of).orElseGet(List::of);
```

Comment:
There is no need to use optional here, we know `iss` is present and instance of Json string